src/Fixer/LanguageConstruct/ErrorSuppressionFixer.php

```diff
@@ -164,8 +164,13 @@ private function isDeprecationErrorCall(Tokens $tokens, $index)
             return false;
         }
 
-        $end = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $tokens->getNextTokenOfKind($index, [T_STRING, '(']));
+        $endBraceIndex = $tokens->findBlockEnd(Tokens::BLOCK_TYPE_PARENTHESIS_BRACE, $tokens->getNextTokenOfKind($index, [T_STRING, '(']));
 
-        return $tokens[$tokens->getPrevMeaningfulToken($end)]->equals([T_STRING, 'E_USER_DEPRECATED']);
+        $prevIndex = $tokens->getPrevMeaningfulToken($endBraceIndex);
```

Comment:
i;m really sad to see this change over and over again :(

Reply:
yeah :( it's similar to checking for leading `\` in functions/classes